pulse-binding/src/context/introspect.rs

```diff
@@ -324,13 +324,23 @@ impl<'a> SinkPortInfo<'a> {
             }
         }
     }
+
+    /// Creates owned data from borrowed data
+    pub fn to_owned(&self) -> SinkPortInfo<'static> {
+        SinkPortInfo {
+            name: self.name.clone().map(|o| Cow::Owned(o.into_owned())),
+            description: self.description.clone().map(|o| Cow::Owned(o.into_owned())),
+            availability_group: self.availability_group.clone().map(|o| Cow::Owned(o.into_owned())),
```

Comment:
You're missing the PA version feature gate for this attribute, hence CI failure.

Otherwise there's no issue with implementing a clone feature for this struct like you have.